build.gradle

```diff
@@ -41,6 +41,8 @@ task coverage(type: JacocoReport) {
 }
 
 dependencies {
+    implementation 'junit:junit:4.13.1'
+    implementation 'junit:junit:4.13.1'
```

Comment:
I think you accidentally placed two similar dependencies here

Reply:
Noted! Will fix!